editor/components/block-list/index.js

```diff
@@ -214,6 +252,35 @@ class BlockList extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { onReplace, onChange } = this.props;
+
+		if ( ! this.shortcutTransforms ) {
+			return;
+		}
+
+		if ( ! isAccess( event ) ) {
+			return;
+		}
+
+		const transform = find( this.shortcutTransforms, ( { shortcut } ) => isAccess( event, shortcut ) );
+
+		if ( ! transform ) {
+			return;
+		}
+
+		const result = transform.transform( this.blocks.map( ( { attributes } ) => attributes ) );
+
+		// Check if we received blocks or attributes.
+		if ( result.uid || Array.isArray( result ) ) {
+			onReplace( this.blocks.map( ( { uid } ) => uid ), castArray( result ) );
```

Comment:
Minor: [Lodash's `_.map`](https://lodash.com/docs/4.17.4#map) property shorthand can be nice for this sort of thing:

```js
map( this.blocks, 'uid' )
```